index.js

```diff
@@ -92,7 +113,14 @@ function findById() {}
  *  filterByGenre(movies, "Horror")
  *  //> []
  */
-function filterByGenre() {}
+function filterByGenre(movies, genre) {
+  // error
+  if (!movies.length) {
+    throw Error
+  }
+  //case insensitive
+  return movies.filter((movie) => movie.genre.toLowerCase().includes(genre.toLowerCase()))
```

Comment:
How could we make sure this filter comes up with only exact matches? i.e., if the search was "war", we would _not_ want to get movies with the genre "heartwarming".